app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,22 @@
+class CommentsController < ApplicationController
+  def create
+    @post = Post.find(params[:post_id])
+    new_comment = current_user.comments.new(
+      text: comment_params,
+      user_id: current_user.id,
+      post_id: @post.id
+    )
+    new_comment.update_comments_counter
+    if new_comment.save
+      redirect_to "/users/#{@post.user_id}/posts/#{@post.id}", notice: 'Success!'
+    else
+      render :new, alert: 'Error occured!'
+    end
+  end
+
+  private
+
+  def comment_params
+    params.require(:comment).permit(:text)[:text]
+  end
+end
```

Comment:
- [ ] The comment text area is only appearing to the two users `Ben and Tailah` while for others it is not appearing. Kindly make sure that all users can see the comment text area.

![image](https://user-images.githubusercontent.com/55635977/154976362-a7331bde-3a45-4313-93d0-4323b41166f4.png)

Reply:
Okay sir